odoo/addons/base/models/res_company.py

```diff
@@ -260,7 +260,7 @@ def write_company_and_print_report(self):
     @api.model
     def action_open_base_onboarding_company(self):
         """ Onboarding step for company basic information. """
-        action = self.env.ref('base.action_open_base_onboarding_company').read()[0]
+        action = self.env.ref('base.action_open_base_onboarding_company').sudo().read()[0]
```

Comment:
Shouldn't all those calls to `read` be replaced by `_for_xml_id`, which seems to exist for that purpose?

Reply:
Not sure, I didn't change the existing calls but we can definitly make it the only access point. In this case maybe find something shorter than `_for_xml_id`